builder/vsphere/supervisor/step_connect_k8s.go

```diff
@@ -0,0 +1,100 @@
+//go:generate packer-sdc struct-markdown
```

Comment:
Consider changing the filename to `step_connect_supervisor.go` (unless we have already settled on k8s in place of SV -- then remove other references of SV).

Reply:
Sounds good, will rename the file and struct define inside as well.